gravitational/gravity
# [5.5.x] Bump monitoring application

## Description

Fixes issue with watcher in monitoring. See [monitoring #184](https://github.com/gravitational/monitoring-app/pull/184) for details.

## Type of change


* Bug fix (non-breaking change which fixes an issue)

## Linked tickets and other PRs



* Requires [monitoring #184](https://github.com/gravitational/monitoring-app/pull/184).

## Implementation


## Testing done

1. Install cluster less than version 5.5.21.
2. Upgrade cluster to the version in this PR.
3. Move old influxdb data to the new path.
4. Check for the errors in the watcher container for influxdb.

## Additional information
